app/src/main/java/com/simplecity/amp_library/utils/SettingsManager.java

```diff
@@ -355,11 +357,15 @@ public int getAccentColor() {
     public static final String KEY_IGNORE_EMBEDDED_ARTWORK = "pref_ignore_embedded_artwork";
     public static final String KEY_IGNORE_FOLDER_ARTWORK = "pref_ignore_folder_artwork";
     public static final String KEY_PREFER_EMBEDDED_ARTWORK = "pref_prefer_embedded";
+    public static final String KEY_PREFER_REMAINING_TIME = "pref_show_remaining_time";
+
 
     public boolean canDownloadArtworkAutomatically() {
         return getBool(KEY_DOWNLOAD_AUTOMATICALLY, false);
     }
 
+    public boolean preferRemainingTime() { return getBool(KEY_PREFER_REMAINING_TIME,true); }
```

Comment:
I would call this `displayRemainingTime`, and change the key to `KEY_DISPLAY_REMAINING_TIME`